frontend/src/redux/reducers/selectors.js

```diff
@@ -0,0 +1,3 @@
+export default {
+    getRestaurantReviews: state => state.restaurantReviewsReducer,
```

Comment:
Do we need this file? Or can we change [this](https://github.com/NickPriv/FryRank/pull/14/files#diff-09ba3a9eb942d8370bdb04c1e336952fb1be0e84fc01ca6e147823a26cf890f1R8) to use this selector instead?

Reply:
I'm deleting it for now.